proposals/NNNN-improve-VDE-for-stable-frame-rate.md

```diff
@@ -0,0 +1,165 @@
+# Improve VirtualDisplayEncoder for stable frame rate
+
+* Proposal: [SDL-NNNN](NNNN-improve-VDE-for-stable-frame-rate.md)
+* Author: [Shinichi Watanabe](https://github.com/shiniwat)
+* Status: **Awaiting review**
+* Impacted Platforms: [Java Suite]
+
+## Introduction
+
+This proposal improve the quality of video projection when using Android `VirtualDisplay` and `MediaEncoder`, as Android `MediaEncoder` does not produce consistent frame rate for rendering `VirtualDisplay's` surface.
```

Comment:
```suggestion
This proposal improves the quality of video projection when using Android `VirtualDisplay` and `MediaEncoder`, as Android `MediaEncoder` does not produce a consistent frame rate for rendering `VirtualDisplay`'s surface.
```

Reply:
Additionally, please make sure all changes to the introduction are also reflected in the PR description.  Thank you!